### ISSUE TITLE
ESLint fails on a project reporting it can't find a configuration file

### ISSUE THREAD

**Issue by akolov** [apexskier/nova-eslint#293]
ESLint Nova was working on that project a few weeks ago, now it doesn't work at all. Running ESLint in Terminal works fine.

**Environment**

- Nova version: `9.1 Build 383286`
- Extension version: `1.7.2`
- ESLint version: `8.12.0`
- macOS version: `12.3.1 (21E258)`
- NodeJS information:
  - node version: `v16.14.0`
  - npm version: `8.3.1`
  - installation method: `nvm`

**Extension console output**

```
ESLint[14:30:01.552000] 
Oops! Something went wrong! :(

ESLint: 8.12.0

ESLint couldn't find a configuration file. To set up a configuration file for this project, please run:

    npm init @eslint/config

ESLint looked for configuration files in /Developer/gated-art-web/components and its ancestors. If it found none, it then looked in your home directory.

If you think you already have a configuration file or if you need more help, please stop by the ESLint chat room: https://eslint.org/chat/help



ESLint[14:30:01.553000] Extension encountered an uncaught exception:
/Users/alex/Library/Application Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js (Line 306, Column 28)
Error: failed to get eslint config for /Developer/gated-art-web/components/TimelinePostContextMenu.vue: 2
    @file:///Users/alex/Library/Application%20Support/Nova/Extensions/apexskier.eslint/Scripts/main.dist.js:306:28
    @[native code]
    promiseReactionJob@[native code]
```

**Additional context**

I use `.eslintrc.yml` as my config file for the project and it's located in the project root. ESLint runs without any problem when invoked from command line.

**Comment by novemberborn** [comment]
@akolov is `/Developer/` meant to be in your user directory? That's what I'm seeing on my end, it's looking in the wrong place.

**Comment by akolov** [comment]
Yep, `/Developer/` is supposed to be in the user directory: `/Users/alex/Developer/` – is the absolute path. It seems that plugin is looking in the root `/Developer` instead?

**Comment by novemberborn** [comment]
There's also https://github.com/apexskier/nova-eslint/pull/283 from @gnattu. I tried to clone and install that but no luck, but I'm relatively new to Nova & extensions.

**Comment by henrikruscon** [comment]
I have the same issue and I also have my projects in the `Developer` folder.

**Comment by ethanhalko** [comment]
Yeah it seems like the extension is looking for configs in `/{Project Path}` instead of `/Users/user/{Project Path}`, additionally it's also looking in `./components` rather than the project root? The only way I've been able to get it to work is by downgrading to Nova 8.

**Comment by kpitt** [comment]
Just encountered this same issue tonight. My project is under `~/Dropbox/...` but path passed to eslint command is `/Dropbox/...`, so I think the `~` representing the home directory is getting stripped.

**Comment by jbrudvik** [comment]
@kpitt I'm also seeing this same behavior -- the leading `~` apparently getting stripped.

**Comment by mindgrub-rkoch** [comment]
I am encountering the same issue, but it looks like it's being handed in #283, so I'll wait for that fix to go in.

**Comment by apexskier** [comment]
Resolved by https://github.com/apexskier/nova-eslint/pull/283

**Comment by apexskier** [comment]
Included in release [v1.7.3](https://github.com/apexskier/nova-eslint/releases/tag/v1.7.3)
